### ISSUE TITLE
Mismatched string formatting when examining critters

### ISSUE THREAD

**Issue by c6-dev** [alexbatalov/fallout1-re#1]
![image](https://user-images.githubusercontent.com/31777460/214910577-ac5e11f9-42de-43ce-8698-d12829e6f197.png)

**Comment by alexbatalov** [comment]
Hey there, are you trying the last build? There was a similar problem which was fixed in b4c378888ef7d5603913896a9ba2c6f915b3d664.

**Comment by c6-dev** [comment]
Yeah, i compiled myself from the latest commit (893a8ea). Will try the latest GHA build to be sure.

**Comment by c6-dev** [comment]
Yep, same problem with the latest github build.

**Comment by alexbatalov** [comment]
Post your save game

**Comment by c6-dev** [comment]
[SLOT06.zip](https://github.com/alexbatalov/fallout1-re/files/10512284/SLOT06.zip)
A slightly different spot from the screenshot, but there is a dead molerat right next to the character.

**Comment by alexbatalov** [comment]
Still no luck, I've it working as expected (just like in vanilla). What exactly is your Fallout version (on the main screen) and where did you get it (GOG, Steam, CD)?

There is difference in `proto.msg` in examine strings (circa 520). In F2 line 522 is "He looks: %s " which acts as formatting string. In my F1 (1.1) line 522 is just "He looks: " which is only a prefix, and formatting string is line 521 ("%s %s"). Somehow you have a mix of styles, I'm wondering where did you get it from.

**Comment by c6-dev** [comment]
Oh, strange. This is a regular GOG version without any alterations. Main screen says 1.1. And proto.msg in DATA/TEXT/ENGLISH/GAME indeed has a mix of styles:
```
{521}{}{%s %s.}
{522}{}{He looks: %s.}
```

**Comment by c6-dev** [comment]
My bad for not checking it first, i do have the same bug when using vanilla executable. 
So there's nothing wrong with RE at least, just a weird mix up in resources.

**Comment by alexbatalov** [comment]
Holy shit, GOG is indeed v1.2 and have both different binary, master.dat and critter.dat.

**Comment by c6-dev** [comment]
I just tested Steam version and got the same bug. But i noticed something strange - if i use the HR executable (falloutwHR.exe) the bug does not occur, but does with regular falloutw.exe and the RE exe.
